modules/ornl/frontier/rct/1.83.0.lua

```diff
@@ -0,0 +1,46 @@
+--[[
+
+    file rct module
+
+    Copyright 2024 RADICAL-Project
+
+]]--
+
+
+-- local vars --
```

Comment:
should we set some RCT env variables by default?
- for all RCT packages it could be `RADICAL_REPORT=TRUE`

p.s., should we set DEBUG log level? or explicitly set INFO, thus it will highlight the possibility to change that env variable? (kinda expose some env variables which are set implicitly within RP/RCT)

Reply:
Thanks, added.